### PR TITLE
Feat video

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
1. Added Video Single Page UI

### CODE REVIEW

File: src/components/customComponents/VideoPlayer.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              <span className="highlight-text icon-text-container">33K</span>
+              <div className="icon-container">
+                <AiFillDislike size={"1.2rem"} color="grey" />
+              </div>
+              <span className="highlight-text icon-text-container"> 929</span>
+              <div className="icon-container">
+                <MdWatchLater size={"1.2rem"} color="grey" />
+              </div>
+              <div className="icon-container">
+                <RiMenuAddFill size={"1.2rem"} color="grey" />
```
Comment (vaibhav18matere) [inline src/components/customComponents/VideoPlayer.js]: you can style all icons at a time; `size=1.2rem` is just repeating in so many places.

**Review by vaibhav18matere — APPROVED** [review]
everything else looks pretty clean..
here's small suggestion, below:

File: src/components/customComponents/VideoPlayer.js (hunk 1 above)
Comment (SimranjitKaurMaan) [reply src/components/customComponents/VideoPlayer.js]: thanks for pointing it out. Corrected it.

**Review by coldpigli — APPROVED** [review]
Everything looks good, no changes suggested

**Pull request merged**